App.js

```diff
@@ -13,6 +13,8 @@ import {styles} from './Appstyle';
 
 const App = () => {
   const [modalVisible, setModalVisible] = useState(false);
+  const [edit, setedit] = useState({});
+  const [select , setSelected] = useState(false);
```

Comment:
Why is this state required?
You can do the task without creating unnecessary states.